contracts/libs/0.8.x/MinterUtils_v0_1_0.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Art Blocks Minter Utils Library
+ * @notice A collection of utility functions that may be used across the Art
+ * Blocks minter ecosystem.
+ * Semantic versioning is used in the solidity file name, and is therefore
+ * controlled by contracts importing the appropriate filename version.
+ * @author Art Blocks Inc.
+ */
+library MinterUtils {
+    /**
+     * @notice Validates that a GenArt721CoreV3 core contract's
+     * `getPrimaryRevenueSplits` function returns the expected number of
+     * return values based on the `isEngine` expected state of the core
+     * contract.
+     * @param isEngine Whether the core contract is expected to be an Art
+     * Blocks Engine contract or not.
+     * @param genArt721CoreAddress The address of the deployed core contract.
+     */
+    function ValidateV3CoreGetPrimaryRevenueSplitsResponse(
```

Comment:
May want to consider making this a `modifier`?

Reply:
@jakerockland - We can't call a modifier in a library from the minter (modifiers may only be called in the current class or a parent class). Additionally, even if we moved this to a parent class, we only know if `_isEngine` is true or false after some logic is performed in the constructor function itself.
We could move this function's logic to a modifier within the library itself, but I don't think that is what you mean here, yeah?

Reply:
Ah – roger that ser, SGTM. In that case, I think implemented as-is makes sense.

> Also note that we could move _isEngine to the minter's specified constructor args, but was thinking it is probably worth the extra on-chain gas during deployment to take some administrative burden off of the deployer and let the minter auto-configure itself as integrating with an engine or flagship core during deployment.

Very much agree with your take here.